# Generated by Django 3.2.18 on 2024-05-28 09:32

import django.core.validators
from django.db import migrations, models
import phonenumber_field.modelfields


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Company',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=100)),
                ('phone', phonenumber_field.modelfields.PhoneNumberField(blank=True, default=None, error_messages={'unique': 'A user with that phone already exists.'}, max_length=128, null=True, region=None)),
                ('email', models.EmailField(max_length=254, null=True, unique=True, validators=[django.core.validators.EmailValidator()], verbose_name='email address')),
                ('logo', models.ImageField(blank=True, null=True, upload_to='employees_logos/')),
                ('bio', models.CharField(blank=True, max_length=100, null=True, verbose_name='Bio')),
                ('created_at', models.DateTimeField(auto_now_add=True, null=True)),
                ('updated_at', models.DateTimeField(auto_now=True, null=True)),
            ],
        ),
    ]
